brave/brave-core
# Notify user that browser start is needed after toggling inline tipping setting

Resolves https://github.com/brave/brave-browser/issues/16941

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Launch browser
- Visit brave://rewards
- Click into Tip Settings
- Toggle one of inline tip options
- Ensure that `Relaunch` button appears in Tip Settings
- Press `Relaunch` and ensure that it relaunches browser and restores current tabs
- Visit an affected inline tip site and verify that it does/doesn't show tip buttons per new setting
